db/migrate/20160805153018_add_pawn_two_square_to_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class AddPawnTwoSquareToPieces < ActiveRecord::Migration
+  def change
+    add_column :pieces, :pawn_two_squares, :boolean
```

Comment:
@jacobwgillespie that is a very good question!  not sure how I forgot to add that logic, but I added it in my latest commit